### ISSUE TITLE
Add country column

### ISSUE THREAD

**Issue by tkw1536** [nilicule/StadiaGameDB#9]
There should be a column (that can be filtered by) which contains the list of countries a game is available in.

**Comment by Mav3r1x** [comment]
Wait! Not all games are available in every country?

**Comment by tkw1536** [comment]
I don’t think this is the case at the moment. However [this page](https://support.google.com/stadia/answer/9566513?hl=en) mentions that "Game availability may vary depending on countries where publishers choose to make their games available". 

I also know that some games are censored in certain countries. For instance, `Wolfenstein: Youngblood` has a separate [German Version](https://stadia.google.com/u/0/store/details/6d92431b6ca24d69a771cf136a2a231frcp1/sku/dfcc2a3f9ab0421c86ba27e35ff8e41a). For comparison the normal version can be found [here](https://stadia.google.com/u/0/store/details/4884bd9bd7864cf28deef05ef4b69e70rcp1/sku/31047b9e82bb4a30ba66b24bef71cdf2). 

Note: I am not sure if the link for the german version will work outside of Germany. Similarly, I am not sure if the link to the "normal" version will work inside Germany.

**Comment by ja1984** [comment]
I just checked and it looks like Kine is the only game that have "Switzerland" listed as a country.

But this looks more like a "miss click" from some one over at google more than anything tho.
